Packs/RetarusSecureEmailGateway/README.md

```diff
@@ -2,8 +2,58 @@
 
 # Retarus Secure Email Gateway
 
-This pack includes Cortex XSIAM content.
+Retarus Secure Email Gateway is a fully managed cloud service that provides comprehensive, multi-layered security for organizations.
 
-Integrate Retarus Secure Email Gateway to seamlessly fetch events from Secure Email Gateway by Retarus and enhance email security.
+It filters all inbound and outbound traffic to defend against threats like malware, ransomware, and phishing using advanced sandboxing technology.
+
+## This pack includes
+
+Data normalization capabilities:
+
+* Data modeling rules normalize Retarus Secure Email Gateway logs that are ingested via the integration to Cortex XSIAM.
+* Ingested logs can be queried in XQL Search using the *`retarus_secure_email_gateway_raw`* dataset.
+
+## Supported log categories
+
+| Category                         | Category Display Name            |
+|:---------------------------------|:---------------------------------|
+| AntiVirus MultiScan Inbound      | MultiScan                        |
+| AntiVirus MultiScan Outbound     | MultiScan                        |
+| Message Transfer Agent Inbound   | MTA                              |
+| Message Transfer Agent Outbound  | MTA                              |
+| CxO Fraud Detection              | CxO                              |
+| Patient Zero Detection           | PZD                              |
+| Sandboxing                       | Sandboxing                       |
+
+### Supported timestamp formats
```

Comment:
If I am not mistaken, this mapping is for an Event Collector- This mention in the README is not required.

In general-
Timestamp parsing will always be done automatically via the event collector, and if not, this should be raised with the relevant developer who wrote the integration.